### DISCUSSION TITLE
Do we have End of Life Roadmap for the Operator versions?

### DISCUSSION THREAD

**Discussion by srujan07** [strimzi/strimzi-kafka-operator#7575] [Q&A]
Hi Team,

We would like to know if we have End of Life Roadmap for the Operator versions? We are currently running 0.27.1v and want to check how long we can get the support for this particular version. It will also help us to plan for upgrades based on EOL. Please share the links/docs(if any) which talks about this.

Thanks in advance,
Srujan.

**Comment by scholzj** [comment]
There is no _support_ for any old Strimzi version. In case of some critical CVEs, we might patch the previous minor release for users who didn't upgraded to the latest version. But we would normally not do any bug fixes. And for really old versions such as 0.27, we would normally not do any patching at all and if you need any bugfix or CVE fix, you should just upgrade to a new version. 

Supporting the old versions is not something what would be feasibel given the effort etc. Also, I think that both Strimzi and Kafka offer relatively good backwards compatibility to make the upgrade sufficiently easy to do.

**Reply by ravishankarsr82** [reply]
Hi,

Is the Strimzi EOL policy still the same? 

Best Regards,
Ravi

**Reply by im-konge** [reply]
Hi, 

currently we have 0.45.x that is supported for longer period of time, because of the support of ZK (which is now removed from the main branch). But we didn't specify the timeline I guess - IIRC it should be around one year, as Kafka supports 3.9.0.
But there are no other EOL policies.
